test/tornado_tests/test_motor_cursor.py

```diff
@@ -516,6 +516,16 @@ async def test_raw_batches(self):
             lst = await method().batch_size(2).to_list(length=1)
             self.assertEqual([{"_id": 0}, {"_id": 1}], bson.decode_all(lst[0]))
 
+    @gen_test
```

Comment:
We should add the same test in [test_asyncio_cursor.py](https://github.com/mongodb/motor/blob/master/test/asyncio_tests/test_asyncio_cursor.py) for completeness